scripts/blog_review.py

```
@@ -0,0 +1,571 @@
+SMART_MAP = {
+    "‘": "'",
+    "’": "'",
+    "‚": "'",
+    "‛": "'",
+    "“": '"',
+    "”": '"',
+    "„": '"',
+    "–": "--",
+    "—": "---",
+    "…": "...",
+    " ": " ",
```

Ruff (0.16.0)

[warning] String contains ambiguous ` ` (NO-BREAK SPACE). Did you mean ` ` (SPACE)?

(RUF001)